nasl/nasl_misc_funcs.c

```diff
@@ -287,7 +289,12 @@ nasl_end_denial (lex_ctxt *lexic)
         }
     }
 
-  retc->x.i_val = 0;
+  // Services seem to not respond.
+  // Last test with boreas
+  if (check_host_still_alive (kb, plug_current_vhost ()))
```

Comment:
mhm, if boreas is either disabled or has an error isn't it better to follow the assumption that it is dead?